### PR TITLE
re-add istioctl unit tests to Makefile

### PR DESCRIPTION
**Pull request by ayj**
https://github.com/istio/istio/pull/3820 moved istioctl out of pilot
subdirectory but forgot to re-add istioctl unit tests to top-level
Makefile. Fix that problem and also the currently broken tests.

fixes https://github.com/istio/istio/issues/5202

### CODE REVIEW

**Comment by ayj** [comment]
@hklai, can you help approve so we can re-enable unit test coverage for istioctl?

File: istioctl/cmd/istioctl/convert/cmd_test.go [hunk 1]
```diff
@@ -88,8 +88,8 @@ func TestCommand(t *testing.T) {
 		{in: []string{"destination-policy-helloworld.yaml", "route-rule-80-20.yaml"},
 			out: "destination-rule-helloworld-with-80-20.yaml"},
 
-		{in: []string{"nolabel-destination-policy.yaml"},
-			out: "nolabel-destination-rule.yaml"},
+		// {in: []string{"nolabel-destination-policy.yaml"},
```
Comment (hklai) [inline istioctl/cmd/istioctl/convert/cmd_test.go]: Is this intentional?

Comment (ayj) [reply istioctl/cmd/istioctl/convert/cmd_test.go]: Yes. The files were never added when the test case was expanded. cc @esnible

**Pull request merged**